base/pkg.jl

```diff
@@ -19,6 +19,37 @@ end
 const cd = Dir.cd
 
 dir(path...) = Dir.path(path...)
+
+"""
+    dir() -> AbstractString
+
+Returns the absolute path of the package directory. This defaults to
+`joinpath(homedir(),".julia","v\$(VERSION.major).\$(VERSION.minor)")` on all platforms (i.e.
+`~/.julia/v0.4` in UNIX shell syntax). If the `JULIA_PKGDIR` environment variable is set,
```

Comment:
maybe this line should be interpolated so this doesn't have to manually be updated each new version?